tests/snmpsim/hpe-msa.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|"HP MSA 2040 SAN"
+1.3.6.1.2.1.1.2.0|6|"DRSAKZR-123456789"
```

Comment:
Not sure to understand, I pushed a PR tell me if it's wrong again. The OIDs should be standardised of the real OID of the device?